contracts/Invest.sol

```diff
@@ -36,6 +37,7 @@ contract Invest is PoolsData {
         uint256 MainCoin; //the amount of the main coin invested (eth/dai), calc with rate
         uint256 TokensOwn; //the amount of Tokens the investor needto get from the contract
         uint256 InvestTime; //the time that investment made
+        uint256 LockedDealId; // the Pool ID of LockedDeal contract
```

Comment:
I think it's better to emit events and not save this (LockedDealId).
the withdraw will be from the locked deal contract